File: src/main/kotlin/ru/umeta/libraryintegration/inmemory/JedisConnector.java
package ru.umeta.libraryintegration.inmemory;

import org.springframework.context.annotation.Bean;
import org.springframework.stereotype.Component;
import redis.clients.jedis.Jedis;

/**
 * Created by k.kosolapov on 3/8/2016.
 */
@Component
public class JedisConnector {
    Jedis jedis = new Jedis("localhost", 6379, 20000);
}

File: src/main/kotlin/ru/umeta/libraryintegration/inmemory/JedisConnector.kt
package ru.umeta.libraryintegration.inmemory

import org.springframework.stereotype.Component
import redis.clients.jedis.Jedis

/**
 * Created by k.kosolapov on 3/8/2016.
 */
@Component
open class JedisConnector {
    internal var jedis = Jedis("localhost", 6379, 20000)
}
